2-mandatory/3-stocks.js

```diff
@@ -34,7 +34,17 @@ const CLOSING_PRICES_LAST_5_DAYS_FOR_ALL_STOCKS = [
         Functions can help with this!
 */
 function getAveragePrices(closingPricesForAllStocks) {
```

Comment:
This is a good attempt! 😄 A few small points:
- The question is expecting us to use the array which is being passed in as a parameter: `closingPricesForAllStocks` but it looks like you're using the `CLOSING_PRICES_LAST_5_DAYS_FOR_ALL_STOCKS` array instead
- Spend some time thinking about the best way to name your variables. For example, in the code below you are using a `prices` variable - but this is really just a single price, so maybe `price` would be a better name here.